ubuntu/common/install_nvidia_fabric_manager.sh

```diff
@@ -8,7 +8,7 @@ nvidia_fabricmanager_distribution=$(jq -r '.distribution' <<< $nvidia_fabricmana
 nvidia_fabricmanager_version=$(jq -r '.version' <<< $nvidia_fabricmanager_metadata)
 nvidia_fabricmanager_sha256=$(jq -r '.sha256' <<< $nvidia_fabricmanager_metadata)
 
-nvidia_fabricmanager_download_url=http://developer.download.nvidia.com/compute/cuda/repos/$nvidia_fabricmanager_distribution/x86_64/nvidia-fabricmanager-${nvidia_fabricmanager_version}_amd64.deb
+nvidia_fabricmanager_download_url=http://developer.download.nvidia.com/compute/cuda/repos/$nvidia_fabricmanager_distribution/x86_64/nvidia-fabricmanager-$nvidia_fabricmanager_version_amd64.deb
```

Comment:
Will this work? how does it know when the variable name ends?

Reply:
Good catch! I will update this

Reply:
Done, thanks Matt!